app/assets/stylesheets/govuk-component/_taxonomy-sidebar.scss

```diff
@@ -19,29 +19,21 @@
     .related-content {
       margin: 0 0 0 15px;
       padding: 0;
-      list-style-type: none;
-      line-height: 1.5;
 
-      // We use a pseudo element to add bullet points to related links. This
-      // makes the bullets clickable and highlights them on link hover. With
-      // this approach, IE8 erroneously applies the anchor tag's underline to
-      // the pseudo element. Specifying separate text-decoration rules for the
-      // anchor tag and a nested span fixes this.
+      li {
+        font-size: 12px;
+      }
+
       a {
         display: block;
-        text-decoration: none;
+        font-size: 16px;
+        line-height: 1.5;
 
-        &:before {
-          content: "•";
-          display: inline-block;
-          width: 15px;
-          margin-left: -15px;
+        span {
```

Comment:
Actually @36degrees I tried to do this another way, but it didn't work so I've left it as-is. I've added a comment to the CSS, repeated here for convenience:

> Since the bullet point has a different font size to the actual link text, the bullet's height is not vertically aligned correctly with the text. Having the text in this span element allows us to move the text relative to the bullet.

I realise this isn't a pretty fix, so if you can think of a better way to do this, I'm all ears.